docs/source/contribution.rst

```diff
@@ -431,15 +431,21 @@ To build the documentation, you need to install `Sphinx <http://www.sphinx-doc.o
 
   $ pip install sphinx sphinx_rtd_theme
 
-Then you can build the documentation in HTML format locally::
+Docstrings (documentation comments in the source code) are collected from the installed Chainer module.
+In order to generate the html files, Chainer must be installed in develop mode. If Chainer has not been installed in develop mode, then ``make html`` will yield an error that the reference sources cannot
+be read.
+
+If you have edited docstrings and want to see those changes reflected in the generated html, it is necessary to reinstall Chainer in develop mode again to see the changes reflected.
```

Comment:
If Chainer is installed in editable mode, I think reinstalling is not needed.

Reply:
You're correct. Confirmed.

Reply:
Actually, reinstallation is often required. For an example, please see this gist: https://gist.github.com/Crissman/d5d60509b678cc2ec15d71d3923b507f#file-gistfile1-txt

Reply:
Not required, and that line removed.